dbms/src/Flash/tests/gtest_aggregation_executor.cpp

```diff
@@ -333,6 +343,20 @@ try
 }
 CATCH
 
+TEST_F(ExecutorAggTestRunner, AggregationSum)
+try
+{
+    // Test output type of the first and second stage sum aggregation function
+    PrecType precision = 33;
+    ScaleType scale = 7;
+    std::shared_ptr<IDataType> input_type_ptr1 = std::make_shared<DataTypeDecimal<Decimal64>>(precision, scale);
```

Comment:
Also test for `Decimal32` , `Decimal128` and `Decimal256` ?

Reply:
> Also test for `Decimal32` , `Decimal128` and `Decimal256` ?

okk